### PR TITLE
Drop-down data switch through Cellular tile.

### PR DESCRIPTION
**Pull request by ashoksoni**
If Sim1 and Sim2 are both from the same providers, options will be SIM1 and SIM2.
Change-Id: I68fa917634ac77178e2f38ae1d0de079e8892488

Signed-off-by: Ashok Soni <ashok.soni@gmail.com>

### CODE REVIEW

File: packages/SystemUI/src/org/codeaurora/internal/IDepersoResCallback.aidl [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2015, The Linux Foundation. All rights reserved.
```
Comment (akhilnarang) [inline packages/SystemUI/src/org/codeaurora/internal/IDepersoResCallback.aidl]: Where are these added from? Seem to be from CAF, if so please add the commits with proper authorship

**Review by akhilnarang — DISMISSED** [review]

**Comment by ashoksoni** [comment]
As I couldn't import package org.codeaurora.internal, I simply copied these files to the required directories. They are from codeaurora vendor and apparently without any authorship. This is done by me so proper authorship.

If there is a simple way to import them, I would gladly remove these files.

**Comment by akhilnarang** [comment]
Alright. Will test and see.

**Comment by akhilnarang** [comment]
Add it to the list of libraries needed in SystemUI's Android.mk, you should be able to import it

**Comment by ashoksoni** [comment]
Removing this one and push a new one.

**Pull request closed without merging**